styles/styles.css

```diff
@@ -0,0 +1,38 @@
+nav{
+    background-color: steelblue;
+    padding: 10px;
+}
+
+nav a {
+    color:white;
+    display: block;
+    padding: 20px;
+}
+
+h1{
+    color: steelblue;
+}
+
+img{
+    width: 150px;
+    height: auto;
+    border: 1px solid steelblue;
+    padding: 2px;
+    box-shadow: 0 0 30px gray;
+} 
+p  { 
+    padding: 10px;
+}
+
+main{
+    max-width: 840px;
+}
+aside{
+    width: 12rem;
+    margin: 2rem;
+    border: 1px solid rgba (0,0,0, 0.2);
```

Comment:
### Error Message
[CSS Validation Service](https://jigsaw.w3.org/css-validator/validator?uri=AndreLinhares-Dev.github.io/wdd130/)
![image](https://github.com/user-attachments/assets/4039b55c-f1a5-4a3b-99f9-3f3a518e406b)

### Solution
The CSS Validation Service is returning an error in your CSS. To resolve this, you simply need to remove the space between the `rgb` function and its parameters. It should look like this: `rgb(0, 0, 0, 0.2);`